tools/FindUnity/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
```

Comment:
it seems kind of an overhead to move this to a C#  project. Is the logic in the bash script really too much that we want to do that?

Reply:
We ultimately have to stop depending on bash, since it's kind of a weird requirement to push on Windows developers.
I'd also rather we not write our build process and non-trivial tools (like this) twice in something terrible like Powershell/cmd for Windows, or something terrible like bash for Mac/Linux. So, my solution is to write it in C# once, and make the entry point to the program be a very simple Powershell/bash script, as appropriate to the platform.